src/index.spec.ts

```diff
@@ -23,9 +515,27 @@ describe('getMatches method', () => {
     expect(result.length).toBe(5);
   });
 
-  it('should return state US matches', () => {
-    const result = getMatches('A1234567', { states: 'NY' });
-    expect(result.length).toBe(1);
+  for (const state of Object.keys(state_fixture)) {
```

Comment:
In general, this is awesome! Thank you! 😄 
I'm wondering if maybe using Jest's [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) could reduce the need for creating tests in nested for loops?

Reply:
I don't think so given that the data we are testing is inherently nested (ie: multiple states, with multiple DL regex patterns which in turn have multiple test cases), I think you would still have nested loops unless you completely flattened the data which might make it more difficult to read.  I figured if the structure of the test data more closely matched the structure of the main data  it would make it easier for people to add new test cases.

Reply:
Thanks!